### PR TITLE
Creating Html field and field editors

### PR DESCRIPTION
**Pull request by sebastienros**

### CODE REVIEW

**Comment by agriffard** [comment]
I managed to edit the subtitle of the blogpost to specify a Multiline editor and edit it.
Will it be set like this in the blog.recipe?.

When I use TheTheme, I can't see the Fields anymore in the detail of the blog post on the front or in the preview .

**Comment by agriffard** [comment]
Fields are well displayed now that the .Location bug is fixed.

File: test/Orchard.Tests/DisplayManagement/Decriptors/DefaultShapeTableManagerTests.cs [hunk 1]
```diff
@@ -384,24 +384,24 @@ public void DefaultPlacementIsReturnedByDefault()
         [Fact]
         public void DescribedPlacementIsReturnedIfNotNull()
         {
-            var shapeDetail = new Shape { Metadata = new ShapeMetadata { DisplayType = "Detail" } };
-            var shapeSummary = new Shape { Metadata = new ShapeMetadata { DisplayType = "Summary" } };
-            var shapeTile = new Shape { Metadata = new ShapeMetadata { DisplayType = "Tile" } };
+            var shapeDetail = new ShapePlacementContext("Foo", "Detail", "", null);
+            var shapeSummary = new ShapePlacementContext("Foo", "Summary", "", null);
+            var shapeTile = new ShapePlacementContext("Foo", "Title", "", null);
```
Comment (agriffard) [inline test/Orchard.Tests/DisplayManagement/Decriptors/DefaultShapeTableManagerTests.cs]: shapeTile => shapeTitle

**Comment by sebastienros** [comment]
update: added markdown field

**Comment by agriffard** [comment]
MarkdownField working.

Placement Alternates fixed. Thanks.

**Pull request merged**